packages/runtime-core/src/components/Suspense.ts

```diff
@@ -244,7 +244,7 @@ function createSuspenseBoundary(
   /* istanbul ignore if */
   if (__DEV__ && !__TEST__ && !hasWarned) {
     hasWarned = true
-    console[console.info ? 'info' : 'log'](
+    console[(console as any).info ? 'info' : 'log'](
```

Comment:
@yyx990803 with 3.9 `console.info` will give an error:
```log
This condition will always return true since the function is always defined.
```

Reply:
It Is Because Type Definition Of  `console.info` In `@types\node` is Not Optional So It Is Always Defined
![image](https://user-images.githubusercontent.com/38488260/81806908-c3922c80-953a-11ea-99ca-4a2497d4616e.png)
### Try Using `// @ts-expect-error`